pkg/config/config.go

```diff
@@ -1140,6 +1140,7 @@ func findUnknownEnvVars(config Config, environ []string) []string {
 	knownVars := map[string]struct{}{
 		// these variables are used by the agent, but not via the Config struct,
 		// so must be listed separately.
+		"DD_INSIDE_CI":      {},
```

Comment:
We need to add this to pkg/flare/envvars.go I think.

Reply:
Good catch. Fixed.